src/astro/sql/__init__.py

```diff
@@ -1,270 +1,10 @@
-from typing import Any, Callable, Iterable, List, Mapping, Optional, Union
-
-import pandas as pd
-
-try:
-    from airflow.decorators.base import TaskDecorator, task_decorator_factory
-except ImportError:
-    from airflow.decorators.base import task_decorator_factory
-    from airflow.decorators import _TaskDecorator as TaskDecorator
-
-from astro.constants import ColumnCapitalization, MergeConflictStrategy
-from astro.sql.operators.append import APPEND_COLUMN_TYPE, AppendOperator
-from astro.sql.operators.cleanup import CleanupOperator
-from astro.sql.operators.dataframe import DataframeOperator
-from astro.sql.operators.drop import DropTableOperator
+from astro.sql.operators.append import AppendOperator, append  # noqa: F401
```

Comment:
```suggestion
from astro.sql.operators.append import AppendOperator, append  # noqa: F401
```
How about we add `# noqa: F401` at the top of the file or in the config for this specific file?

Reply:
There isn't a way in flake8 to ignore a block of code as per the maintainer of flake8

Your options are:

1. ignore each line that produces an error by putting # noqa: E501 on it
2. ignore the entire file (but this turns off all other errors as well) with a # flake8: noqa on a line by itself
3. ignore E501 in the entire file by using per-file-ignores:
```ini
[flake8]
per-file-ignores =
     path/to/file.py: E501
```

Reply:
will do the 3rd option

Reply:
Done in https://github.com/astronomer/astro-sdk/pull/637/commits/98fd8a561cb136b865525b33b5b1207dffd1d122